src/astro/sql/operators/transform.py

```diff
@@ -0,0 +1,397 @@
+import inspect
+from typing import Callable, Dict, Iterable, Mapping, Optional, Tuple, Union
+
+import pandas as pd
+from airflow.decorators.base import DecoratedOperator, task_decorator_factory
+from airflow.exceptions import AirflowException
+from sqlalchemy.sql.functions import Function
+
+from astro.databases import create_database
+from astro.sql.table import Table
+
+
+class TransformOperator(DecoratedOperator):
+    """Handles all decorator classes that can return a SQL function"""
+
+    # todo: Add docstrings
```

Comment:
Can we handle this in this PR itself please

Reply:
@kaxil removed this line. there are docstrings for all functions.